sotodlib/preprocess/processes.py

```diff
@@ -779,6 +779,50 @@ def process(self, aman, proc_aman):
                 subtract_name = self.process_cfgs["subtract_name"]
                 )
 
+class A2Stats(_Preprocess):
+    """
+    Calculate statistical metrics for A2, the 2f-demodulated Q and U signals.
+
+    Example config block::
+
+      - name: "a2_stats"
+        calc:
+          stat_names: ["mean", "median", "var", "ptp"]
+          subscan: True
+        save: True
+
+    """
+    name = "a2_stats"
+
+    def calc_and_save(self, aman, proc_aman):
+        # Get A2 signal using the demod_tod function; this function generates intermediary
+        # dsT, demodQ, demodU fields in the AxisManager that will be deleted
+        hwp.demod_tod(aman, demod_mode=2)
+
+        # Compute A2 stats
+        stat_names = self.calc_cfgs.get("stat_names", ["mean", "median", "var", "ptp"])
+        split_subscans = self.calc_cfgs.get("subscan", False)
+        am_name = "a2_stats"
+        # Q
+        tod_ops.flags.get_stats(aman, aman.demodQ, stat_names=stat_names, split_subscans=split_subscans, name=am_name, merge=True)
```

Comment:
Let's not `merge=True` see some of the other processes for an example but create a new axis_manager and wrap all saved data products into that axis_manager then send that to `save`